todo_app/data/classes/ViewModel.py

```diff
@@ -0,0 +1,19 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def not_started_items(self):
+        item_is_not_started = lambda item : item.status == 'Not Started'
+        not_started_items = list(filter(item_is_not_started, self._items))
```

Comment:
Definitely an opinion not a recommendation, but I always find filter a bit ugly and stick to list comprehensions, but it appears I've stumbled upon [a bit of a minefield](https://stackoverflow.com/questions/3013449/list-comprehension-vs-lambda-filter) looking for any form of justification!

Reply:
I'm convinced by your opinion and some of the justifications given on that Stack Overflow link about similarity to natural language/readability, so I've changed this.